.github/workflows/main.yml

```diff
@@ -37,10 +37,10 @@ jobs:
           java-version: '17'
 
       - name: Install Vulkan SDK
-        uses: humbletim/install-vulkan-sdk@v1.1.1
+        uses: humbletim/setup-vulkan-sdk@523828e49cd4afabce369c39c7ee6543a2b7a735
```

Comment:
This seems like it should be ok.  

We just need the Vulkan SDK installed on the build machine (because we don't use a dependencies repo anymore).

If this is the right fix to that then i'm good with it.